### PR TITLE
Lift numberList state up

### PR DESCRIPTION
**Pull request by wanecek**
Lift state of numberList up one level. This comes with two benefits, none which we make use of yet:
1. The option to pass numberList, or methods making use thereof, to the AdminBar (and the Header, I guess). This is probably the easiest way to enable #25, at least that I can think of right now.
1. Enables unit-testing of NumberContainer - but we don't have any test-suite anyway... :man_shrugging:

### CODE REVIEW

**Comment by wanecek** [comment]
I haven't tested this properly, and there are a ton of ways of achieving the same result (Higher order components, Context API, redux etc). Submitting the PR even though it's not completely thought through to get your opinion on this, @OskarDamkjaer.

File: front-end/src/Numbers/NumberListFetcher.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+class NumberListFetcher extends React.Component {
```
Comment (wanecek) [inline front-end/src/Numbers/NumberListFetcher.js]: Maybe not the best name :thinking:

File: front-end/src/Numbers/NumberListFetcher.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  componentDidMount () {
+    const eventSource = new EventSource('http://localhost:3000/subscribe')
+
+    eventSource.onmessage = e => {
+      const numberList = JSON.parse(e.data)
+      if (numberList) this.setState({ numberList })
+    }
+
+    //TODO kolla detta
+    eventSource.addEventListener('eventType', e => {})
```
Comment (wanecek) [inline front-end/src/Numbers/NumberListFetcher.js]: Should we be closing the eventSource as well?

**Comment by wanecek** [comment]
Rebased :+1: :smile:

File: front-end/src/Numbers/NumberListFetcher.js (hunk 2 above)
Comment (OskarDamkjaer) [reply front-end/src/Numbers/NumberListFetcher.js]: The eventlistener is useless and should be removed :)

**Pull request merged**